tasks/task1.js

```diff
@@ -0,0 +1,96 @@
+// Task 1
+function summ(...args) {
+  let sum = 0;
+  for (let i = 0; i < args.length; i += 1) {
+    if (+args[i] && typeof args[i] !== 'boolean') {
+      sum += +args[i];
+    }
+  }
+  return sum;
+}
+
+summ(1, '2', '3', 5, 'abc');
+
+function summAdvanced(...args) {
```

Comment:
Try to simplify (less code)

Reply:
I replaced it with foreach, but it didn't get shorter :)